src/automations/components/AutomationActionDescriptionRunDeployment.vue

```diff
@@ -42,8 +52,8 @@
     action: AutomationActionRunDeployment,
   }>()
 
-  const { value: showParametersModal, setTrue: openParametersModal } = useBoolean()
-  const { value: showJobVariablesModal, setTrue: openJobVariablesModal } = useBoolean()
+  const { value: showParametersModal, setTrue: openParametersModal, setFalse: closeParametersModal } = useBoolean()
```

Comment:
huh. i haven't thought of using this over `useShowModal` but i suppose they're the same.. 

seems confusing. y'know from a "one way of doing things" sort of way

Reply:
oh dang, I forgot that existed 😂 I should just use that. I feel like `useShowModal` existed and then `useBoolean` came around because we wanted something more generic. Should I change this or does it not really matter?

Reply:
I updated it cause now that you mentioned it it started bothering me.

Reply:
i'm on the fence between the 2, but would err toward consistency and least surprise / most-familiar.

I _think_ that'd be useShowModal to me. reading `useBoolean` in a vacuum is surprisingly not immediately intuitive to me what it does, but i get how we prob got here lol. def nbd tho - 🤷